git-workflow/createPB.sh

```diff
@@ -7,113 +7,137 @@
 # project version is on the form 5.1.x-acc210-SNAPSHOT on the form 
```

Comment:
* the news and wallet addons are missing
* the project dependency replacements are missing for the addons added in 5.3